--- splix-1.0.1/src/spl2.cpp	2007-02-04 02:44:20.000000000 +0100
+++ splix-1.0.1-1/src/spl2.cpp	2007-02-10 15:20:18.000000000 +0100
@@ -168,7 +168,8 @@ int SPL2::printPage(Document *document, 
     header[0xd] = _printer->docHeaderValues(1);     // ??? XXX
     header[0xe] = _printer->qpdlVersion();          // QPDL Version
     header[0xf] = _printer->docHeaderValues(2);     // ??? XXX
-    if (_printer->resolutionY() != _printer->resolutionX())
+    if (_printer->resolutionY() != _printer->resolutionX() || 
+        _printer->docHeaderValues(2) == 1)
         header[0x10] = _printer->resolutionX() / 100; // X Resolution
     else
         header[0x10] = 0;                           // X Resolution = Y Res.
