--- src/webkit/webkit.c	2008-03-18 22:58:00.000000000 +0200
+++ src/webkit/webkit.c	2009-02-07 11:04:37.000000000 +0200
@@ -67,7 +67,12 @@
 		g_ascii_strcasecmp (content_type, "application/xhtml+xml") == 0
 		? "application/xhtml"
 		: content_type;
-	webkit_web_view_load_string (WEBKIT_WEB_VIEW (htmlwidget), string, content_type, "UTF-8", base);
+		
+#if WEBKIT_MINOR_VERSION > 0
+	webkit_web_view_load_html_string (WEBKIT_WEB_VIEW (htmlwidget), string, "file://");
+#else
+	webkit_web_view_load_string (WEBKIT_WEB_VIEW (htmlwidget), string, content_type, "UTF-8", "file://");
+#endif
 }
 
 static void
