diff -urN barry-0.14.orig/examples/addcalendar.cc barry-0.14/examples/addcalendar.cc
--- barry-0.14.orig/examples/addcalendar.cc	2008-09-25 09:27:53.000000000 +0800
+++ barry-0.14/examples/addcalendar.cc	2008-11-18 17:55:47.000000000 +0800
@@ -25,6 +25,7 @@
 #include <string.h>
 #include <barry/barry.h>
 #include <iostream>
+#include <cstring>
 
 using namespace std;
 using namespace Barry;
diff -urN barry-0.14.orig/src/controller.cc barry-0.14/src/controller.cc
--- barry-0.14.orig/src/controller.cc	2008-09-25 09:27:53.000000000 +0800
+++ barry-0.14/src/controller.cc	2008-11-18 17:58:38.000000000 +0800
@@ -27,6 +27,9 @@
 #include "data.h"
 #include "endian.h"
 #include <string.h>
+#include <sstream>
+#include <iomanip>
+#include <cstring>
 
 #define __DEBUG_MODE__
 #include "debug.h"
diff -urN barry-0.14.orig/src/data.cc barry-0.14/src/data.cc
--- barry-0.14.orig/src/data.cc	2008-09-25 09:27:53.000000000 +0800
+++ barry-0.14/src/data.cc	2008-11-18 17:59:06.000000000 +0800
@@ -27,6 +27,8 @@
 #include <stdexcept>
 #include <string.h>
 #include <stdlib.h>
+#include <cstring>
+#include <cstdlib>
 
 //#define __DEBUG_MODE__
 #include "debug.h"
diff -urN barry-0.14.orig/src/ldif.cc barry-0.14/src/ldif.cc
--- barry-0.14.orig/src/ldif.cc	2008-09-25 09:27:53.000000000 +0800
+++ barry-0.14/src/ldif.cc	2008-11-18 17:59:17.000000000 +0800
@@ -26,6 +26,7 @@
 #include <iostream>
 #include <iomanip>
 #include <string.h>
+#include <cstring>
 
 #define __DEBUG_MODE__
 #include "debug.h"
diff -urN barry-0.14.orig/src/packet.cc barry-0.14/src/packet.cc
--- barry-0.14.orig/src/packet.cc	2008-09-25 09:27:53.000000000 +0800
+++ barry-0.14/src/packet.cc	2008-11-18 17:59:26.000000000 +0800
@@ -31,6 +31,7 @@
 #include "builder.h"
 #include "error.h"
 #include <string.h>
+#include <cstring>
 
 #define __DEBUG_MODE__
 #include "debug.h"
diff -urN barry-0.14.orig/src/r_calendar.cc barry-0.14/src/r_calendar.cc
--- barry-0.14.orig/src/r_calendar.cc	2008-09-25 09:27:53.000000000 +0800
+++ barry-0.14/src/r_calendar.cc	2008-11-18 17:59:40.000000000 +0800
@@ -32,6 +32,7 @@
 #include <time.h>
 #include <string.h>
 #include <stdexcept>
+#include <cstring>
 
 #define __DEBUG_MODE__
 #include "debug.h"
diff -urN barry-0.14.orig/src/record.cc barry-0.14/src/record.cc
--- barry-0.14.orig/src/record.cc	2008-09-25 09:27:53.000000000 +0800
+++ barry-0.14/src/record.cc	2008-11-18 18:00:00.000000000 +0800
@@ -37,6 +37,7 @@
 #include <time.h>
 #include <string.h>
 #include <stdexcept>
+#include <cstring>
 
 #define __DEBUG_MODE__
 #include "debug.h"
diff -urN barry-0.14.orig/src/r_task.cc barry-0.14/src/r_task.cc
--- barry-0.14.orig/src/r_task.cc	2008-09-25 09:27:53.000000000 +0800
+++ barry-0.14/src/r_task.cc	2008-11-18 17:59:48.000000000 +0800
@@ -29,6 +29,7 @@
 #include <ostream>
 #include <iomanip>
 #include <string.h>
+#include <cstring>
 
 using namespace std;
 using namespace Barry::Protocol;
diff -urN barry-0.14.orig/src/socket.cc barry-0.14/src/socket.cc
--- barry-0.14.orig/src/socket.cc	2008-09-25 09:27:53.000000000 +0800
+++ barry-0.14/src/socket.cc	2008-11-18 18:00:29.000000000 +0800
@@ -30,6 +30,7 @@
 #include "sha1.h"
 #include <sstream>
 #include <string.h>
+#include <cstring>
 
 using namespace Usb;
 
diff -urN barry-0.14.orig/tools/btranslate.cc barry-0.14/tools/btranslate.cc
--- barry-0.14.orig/tools/btranslate.cc	2008-09-25 09:27:53.000000000 +0800
+++ barry-0.14/tools/btranslate.cc	2008-11-18 18:01:03.000000000 +0800
@@ -20,6 +20,9 @@
 #include <string.h>
 #include <stdlib.h>
 #include <limits.h>
+#include <cstdlib>
+#include <cstring>
+#include <climits>
 
 using namespace std;
 
