--- old	2007-05-28 19:45:00.000000000 -0400
+++ SConstruct	2007-05-28 19:45:48.000000000 -0400
@@ -14,7 +14,7 @@ ipager_options.AddOptions(
 
     BoolOption('xinerama', 'support xinerama', 0),
 
-    PathOption('PREFIX', 'install-path base', '/usr/local'),
+    PathOption('PREFIX', 'install-path base', '/usr'),
     PathOption('DESTDIR', 'install to $DESTDIR/$PREFIX', '/')
 )
 
@@ -115,7 +115,6 @@ else:
     else:
         print "yes"
         ipager_env.AppendUnique(
-            CPPPATH = imlib2_env.Dictionary()['CPPPATH'], 
             CCFLAGS = imlib2_env.Dictionary()['CCFLAGS'], 
             LIBPATH = imlib2_env.Dictionary()['LIBPATH'],
             LIBS    = imlib2_env.Dictionary()['LIBS']
