--- src/Makefile.orig	2007-03-23 23:05:05.000000000 +0100
+++ src/Makefile	2007-03-23 23:05:40.000000000 +0100
@@ -5,7 +5,7 @@
 #
 
 CXXFLAGS	+= -I../include -Wall -g -O0
-LDFLAGS		+= -lcups -lcupsimage
+LDADD		+= -lcups -lcupsimage
 
 OBJECTS		:= spl2.o printer.o band.o compress.o bandanalyser.o
 HEADERS		:= include/spl2.h include/document.h include/printer.h \
@@ -15,10 +15,10 @@
 
 all: rastertospl2
 rastertospl2: $(OBJECTS) rastertospl2.o raster.o
-	$(CXX) $(LDFLAGS) -o $@ $^
+	$(CXX) $(LDFLAGS) -o $@ $^ $(LDADD)
 
 pbmtospl2: $(OBJECTS) pbmtospl2.o pbmimage.o
-	$(CXX) $(LDFLAGS) -o $@ $^
+	$(CXX) $(LDFLAGS) -o $@ $^ $(LDADD)
 
 %.o: %.cpp $(HEADERS)
 	$(CXX) $(CXXFLAGS) -c $<
