--- ./src/notify.py.orig	2009-09-21 19:06:21.000000000 +0400
+++ ./src/notify.py	2009-09-21 19:07:21.000000000 +0400
@@ -593,7 +593,7 @@
 	def version_reply_handler(self, name, vendor, version, spec_version=None):
 		if spec_version:
 			version = spec_version
-		elif vendor == 'Xfce' and version == '0.1.0':
+		elif vendor == 'Xfce' and version.startswith('0.1.0'):
 			version = '0.9'
 		version_list = version.split('.')
 		self.version = []
