diff -Nurd gajim-0.11/src/common/helpers.py gajim-0.11-modified/src/common/helpers.py
--- gajim-0.11/src/common/helpers.py	2006-11-28 21:34:07.000000000 -0500
+++ gajim-0.11-modified/src/common/helpers.py	2007-02-14 11:54:15.000000000 -0500
@@ -764,14 +764,15 @@
 	check OUR status and if we allow notifications for that status
 	type is the option that need to be True e.g.: notify_on_signing
 	is_first_message: set it to false when it's not the first message'''
-	if advanced_notif_num is not None:
-		popup = gajim.config.get_per('notifications', str(advanced_notif_num),
-			'popup')
-		if popup == 'yes':
-			return True
-		if popup == 'no':
-			return False
-	if type and (not gajim.config.get(type) or not is_first_message):
+#	if advanced_notif_num is not None:
+#		popup = gajim.config.get_per('notifications', str(advanced_notif_num),
+#			'popup')
+#		if popup == 'yes':
+#			return True
+#		if popup == 'no':
+#			return False
+        
+	if type and (not gajim.config.get(type)):
 		return False
 	if gajim.config.get('autopopupaway'): # always show notification
 		return True
diff -Nurd gajim-0.11/src/conversation_textview.py gajim-0.11-modified/src/conversation_textview.py
--- gajim-0.11/src/conversation_textview.py	2006-12-03 07:23:44.000000000 -0500
+++ gajim-0.11-modified/src/conversation_textview.py	2007-02-14 11:54:15.000000000 -0500
@@ -84,6 +84,9 @@
 		color = gajim.config.get('statusmsgcolor')
 		self.tagStatus.set_property('foreground', color)
 
+                self.tagNick = buffer.create_tag('nick')
+                self.tagNick.set_property('weight', pango.WEIGHT_BOLD)
+
 		colors = gajim.config.get('gc_nicknames_colors')
 		colors = colors.split(':')
 		for color in xrange(len(colors)):
@@ -779,8 +782,9 @@
 		if name:
 			buffer = self.tv.get_buffer()
 			end_iter = buffer.get_end_iter()
-			name_tags = other_tags_for_name[:] # create a new list
+                        name_tags = other_tags_for_name[:] # create a new list
 			name_tags.append(kind)
+                        name_tags.append('nick')
 			before_str = gajim.config.get('before_nickname')
 			before_str = helpers.from_one_line(before_str)
 			after_str = gajim.config.get('after_nickname')
